packages/embark/src/lib/modules/storage/index.js

```diff
@@ -41,7 +41,7 @@ class Storage {
   }
 
   addSetProviders(cb) {
-    let code = `\nEmbarkJS.Storage.setProviders(${JSON.stringify(this.storageConfig.dappConnection || [])});`;
+    let code = `\nEmbarkJS.Storage.setProviders(${JSON.stringify(this.storageConfig.dappConnection || [])}, {web3});`;
```

Comment:
Swarm requires a web3 instance now? Is it only for a new version? Does it break old versions?

Reply:
NVM, I just saw that we used to use the global web3.